lib/manageiq/api/common/headers.rb

```diff
@@ -11,6 +11,15 @@ def self.current
           Thread.current[:attr_current_headers]
         end
 
+        def self.encode(val)
```

Comment:
@syncrou Who would use this, we are not building any new headers we are just forwarding headers we receive from 3scale to other services

Reply:
@Fryguy asked for this on a Topology PR, at least to have a helper method to handle it. I put it in  Headers so as to not lose that conversation.  Do you see a need for `self.encode` @Fryguy?

Reply:
I had asked for a common "encode this specific identity structure and put it in a header" method.  I don't see a need for exposing the generic encoding of any arbitrary hash.